### ISSUE TITLE
Escape key does not close bezel

### ISSUE THREAD

**Issue by adamalton** [snark/jumpcut#81]
#### Steps:

1. Copy some things so that you've got some entries in the history.
2. Press and hold your keyboard shortcut to bring up the bezel, e.g. `cmd + shift + v`.
3. Release the `v` key and (while still holding the modifier keys) press `esc`.

#### Expected behaviour
The bezel closes and nothing is pasted.

#### Actual behaviour
The bezel is not closed. Releasing the modifier keys pastes the current item.

In other words, once you've opened the bezel, there's no way to cancel it, you **have** to paste something.

This used to work, but seems to have regressed in a recent release. It's really useful to be able to see what's in your clipboard without pasting it, or to be able to check whether or not you copied something by looking in the bezel, but after this recent regression it's no longer possible.

**Comment by snark** [comment]
@adamalton Definitely a regression, definitely annoying! This was reported as issue #77 -- I've was unable to track down the root cause to my satisfaction, but there is a fix in the main branch and a release will be arriving with the fix sometime this week. Thanks for using Jumpcut, and thanks for reporting the issue!

**Comment by adamalton** [comment]
Ah, nice, thanks! Sorry, I did look for an existing issue, but clearly not hard enough!

**Comment by snark** [comment]
No worries -- bug reports are genuinely appreciated.
